src/components/Home.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable max-len */
```

Comment:
- [ ] Kindly remove the eslint disable rule here. It's best to fix the issue it makes the code more tidy and professional.

Reply:
Well done